packages/node/src/awaiter.ts

```diff
@@ -0,0 +1,27 @@
+export class Awaiter {
```

Comment:
Leave a brief comment to describe what this class does so that readers don't have to read the whole code to grasp the overview.